src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,37 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.Person;
+
+/**
+ * Association class that keeps track of addition and deletion
+ * of a tag for a specific person during a session
+ */
+public class Tagging {
+	
+	private Person person;
+	private Tag tag;
+	private boolean isAddition;
```

Comment:
A boolean variable works here, but an `enum` may be more appropriate because the two values we need are `added` and `deleted`, not `true` and `false`.